components/global/footer.tsx

```diff
@@ -14,16 +14,48 @@ const footerLinks = [
   },
 ];
 
+const footerIcons = [
+  {
+    label: "Twitter",
+    url: "https://x.com/hypercerts",
+    icon: "/socials_icons/twitter-logo.png",
+  },
+  {
+    label: "Discord",
+    url: "https://discord.gg/VVSyKg75",
+    icon: "/socials_icons/discord-logo.png",
+  },
+  {
+    label: "Hypercerts Foundation",
+    url: "/",
+    icon: "/hypercerts-logo.png",
+  },
+  {
+    label: "Github",
+    url: "https://github.com/hypercerts-org/",
+    icon: "/socials_icons/github-logo.png",
+  },
+  {
+    label: "Telegram",
+    url: "https://hypercerts.org/docs/",
+    icon: "/socials_icons/telegram-logo.png",
+  },
+];
+
 const Footer = () => (
   <footer className="flex flex-col space-y-2 justify-center items-center py-5 px-3 border-t-[1.5px] border-black pb-24 md:pb-5 bg-slate-50 mt-auto">
-    <Link className="relative" href={"/"} title="Hypercerts">
-      <Image
-        src={"/hypercerts-logo.png"}
-        alt="Hypercerts logo"
-        width={48}
-        height={48}
-      />
-    </Link>
+    <ul className="flex space-x-4">
+      {footerIcons.map((item) => (
+        <Link
+          className="relative"
+          href={item.url}
+          title={item.label}
+          key={item.label}
+        >
+          <Image src={item.icon} alt={item.label} width={48} height={48} />
+        </Link>
```

Comment:
If these are all external links, you want to use something other than the Next.js Link element. The Link element is specified to use for the Next.js router.  You should probably just be using an `<a>` tag